src/components/organism/SwapTokens/index.tsx

```diff
@@ -505,13 +517,24 @@ const SwapTokens = () => {
     dispatch({ type: ActionType.SET_SWAP_FINALIZED, payload: false });
   };
 
+  const onSwapSelectModal = (tokenData: any) => {
+    setSelectedTokens((prev) => {
+      return {
+        ...prev,
+        [tokenSelectionModal]: tokenData,
+      };
+    });
+  };
+
   useEffect(() => {
-    if (inputEdited.inputType === InputEditedType.exactIn && selectedTokenAValue.tokenValue > 0) {
-      tokenAValue(selectedTokenAValue.tokenValue);
-    } else if (inputEdited.inputType === InputEditedType.exactOut && selectedTokenBValue.tokenValue > 0) {
+    if (tokenSelected.tokenSelected === "A" && selectedTokenBValue.tokenValue > 0) {
```

Comment:
Instead of "A" use enum 
enum TokenPosition {
  tokenA = "A",
  tokenB = "B",
}